qiskit/max-cut/maxCut_qiskit.py

```diff
@@ -31,6 +31,25 @@
 input_path = sys.argv[1]
 output_path = sys.argv[2]
 
+# To include the weight information, we'll need to map the weight to the label
+# so the parser can read it. Therefore, remove all existing "label" lines and
+# replace all "weight" lines with "label" lines.
+with open(input_path, 'r+') as file:
+    # Read all lines from the file
+    lines = file.readlines()
+
+    # Move the file cursor to the beginning
+    file.seek(0)
+
+    # Iterate through the lines, removing lines with "label" and replacing "weight" with "label"
+    for line in lines:
+        if "label" not in line:
+            modified_line = line.replace("weight", "label")
+            file.write(modified_line)
+
+    # Truncate the remaining content in case the new content is shorter than the old content
+    file.truncate()
+
```

Comment:
This seems a bit unnecessary. Why don't you just use the `weight` attribute?

Reply:
The parser library that we use apparently can just parse the label but no custom attributes like our weight.

Reply:
That's weird :/